Python/Product/VSInterpreters/Interpreter/PythonRegistrySearch.cs

```diff
@@ -33,6 +33,15 @@ class PythonRegistrySearch {
         public const string CompanyPropertyKey = "Company";
         public const string SupportUrlPropertyKey = "SupportUrl";
 
+        // When Python is installed as a Store app, it does not expose
+        // the normal registry keys (because Store apps aren't allowed
+        // to write to the regular registry). It does expose the key
+        // when queried directly (e.g. as HKCU\Software\Python\PythonCore\3.7),
+        // but enumerating HKCU\Software\Python\PythonCore key does not
+        // find it (because of deliberate limitations in Windows).
+        private static readonly string[] StoreAppTags = new[] { "3.7", "3.8", "3.9" };
```

Comment:
What about Python 3.6? 2.6? 2.7?

Reply:
We'll never have Windows store versions of those.